### PR TITLE
(CI) Fix SHARK benchmark workflow.

### PR DESCRIPTION
**Pull request by monorimet**

### CODE REVIEW

**Comment by pzread** [comment]
I guess in `.github/workflows/run_shark_tank.yml`, `if: needs.setup.outputs.should-run == 'true'` needs to be changed to `if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'benchmark_cpu')`. Same to the `benchmark_gpu`.

Otherwise the benchmark jobs won't run: https://github.com/openxla/iree/actions/runs/6899590883

This is due to the changes in `configure_ci.py` by https://github.com/openxla/iree/pull/14312

File: .github/workflows/run_shark_tank.yml
Comment (pzread) [inline .github/workflows/run_shark_tank.yml:1]: Sorry I missed in the first pass. `should-run: ${{ steps.configure.outputs.should-run }}` in `setup` job's `outputs` also needs to be replaced with:

```
enabled-jobs: ${{ steps.configure.outputs.enabled-jobs }}
```

**Comment by monorimet** [comment]
> I guess in `.github/workflows/run_shark_tank.yml`, `if: needs.setup.outputs.should-run == 'true'` needs to be changed to `if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'benchmark_cpu')`. Same to the `benchmark_gpu`.
> 
> Otherwise the benchmark jobs won't run: https://github.com/openxla/iree/actions/runs/6899590883
> 
> This is due to the changes in `configure_ci.py` by #14312

Thank you. With these changes, it seems though the benchmark_cpu / benchmark_cuda jobs are no longer skipped, they fail without useful information. Have you seen this sort of failure before? https://github.com/openxla/iree/actions/runs/6936765082

File: .github/workflows/run_shark_tank.yml
Comment (monorimet) [reply .github/workflows/run_shark_tank.yml:1]: Thanks! This seems to have done the trick.

**Comment by monorimet** [comment]
~~Does the runtime build command `pip install -e runtime` need to be run from the IREE source directory? it seems to be looking for a setup script (to build) from the `iree-build` directory.~~
https://github.com/openxla/iree/actions/runs/6937219853/job/18870855644?pr=15630#step:4:11362

**Comment by monorimet** [comment]
It seems that running the pip install is to install the (already built) compiler/runtime as pip packages, so disregard the previous comment. I'll see what the pip install usually picks up for setup.

**Comment by monorimet** [comment]
Fixed setup issue. Need to see where we lost some models in the testing suite, but hopefully this is otherwise ready to go

**Comment by benvanik** [comment]
Pretty sure this is stale now; closing.

**Pull request closed without merging**